config.yaml

```diff
@@ -2001,8 +2001,8 @@ params:
     gif: ""
   static_url: "http://localhost:1313/"
   announcement_banner:
-    text: "<strong>Logging</strong> is here!"
-    link: "blog/announcing-logs/"
+    text: "The <strong>Service Map</strong> for APM is here!"
+    link: "blog/service-map/"
```

Comment:
the link should redirect to the doc page, not the blog article:

https://docs.datadoghq.com/tracing/servicemap/